compiler/optimizer/FieldPrivatizer.cpp

```diff
@@ -85,6 +85,15 @@ TR_FieldPrivatizer::TR_FieldPrivatizer(TR::OptimizationManager *manager)
 
 int32_t TR_FieldPrivatizer::perform()
    {
+
+   // Need to confirm places calling opCodeFor* APIs are doing the right
+   // thing for readbar and wrtbar. All the transformations applied to normal
+   // load/write s should be applied to rd/wrtbar s. However, we need to be
+   // careful about the difference in the shape of the trees and the
+   // mapping relationship between different loads and writes
+   if (comp()->mayContainUnsupportedReadbarOrWritebar())
```

Comment:
Where is the code that returns true for this query ? The implementation in this commit returns false.

Reply:
the subprojects will overwrite the answer to return true depending on options enabled.